java/kafka/streams/src/main/java/KafkaStreamsExample.java

```diff
@@ -39,11 +39,7 @@ public static void main(String[] args) {
 
         TracingSystem tracingSystem = config.getTracingSystem();
         if (tracingSystem != TracingSystem.NONE) {
-            if (tracingSystem == TracingSystem.JAEGER) {
-
-                KafkaClientSupplier supplier = new  io.opentracing.contrib.kafka.streams.TracingKafkaClientSupplier(TracingInitializer.jaegerInitialize());
-                streams = new KafkaStreams(builder.build(), props, supplier);
-            } else if (tracingSystem == TracingSystem.OPENTELEMETRY) {
+            if (tracingSystem == TracingSystem.OPENTELEMETRY) {
```

Comment:
I think you should also remove the Jaeger related dependencies in the Streams application pom.xml file.